python/ray/tune/api_v2/tuner.py

```diff
@@ -0,0 +1,345 @@
+import copy
```

Comment:
This file has a bunch of classes right now. Could we have it hold only the Tuner class for readability?

Reply:
split into tuner_internal.py, tuner.py and tuner_config.py